snorkel/learning/gen_learning.py

```diff
@@ -314,35 +241,39 @@ class prior (in log scale)
 
         self.fg = fg
         self.nlf = n
+        self.cardinality = cardinality
 
     def learned_lf_stats(self):
         """
         Provides a summary of what the model has learned about the labeling
-        functions. For each labeling function, estimates of the following 
+        functions. For each labeling function, estimates of the following
         are provided:
 
+            Abstain
+            Accuracy
+            Coverage
+
+            [Following are only available for binary tasks]
             True  Positive (TP)
             False Positive (FP)
             True  Negative (TN)
             False Negative (FN)
-            Abstain
-
-            Accuracy
-            Coverage
 
         WARNING: This uses Gibbs sampling to estimate these values. This will
                  tend to mix poorly when there are many very accurate labeling
                  functions. In this case, this function will assume that the
                  classes are approximately balanced.
         """
         if self.fg is None:
-            raise ValueError("Must fit model with train() before computing diagnostics.")
+            raise ValueError(
+                "Must fit model with train() before computing diagnostics.")
 
         burnin = 500
```

Comment:
This runs pretty quickly (for test_categoricals.py, it's about 0.6 seconds).

The key point is that the factor graph at this point is reduced to just 1 data point (https://github.com/HazyResearch/snorkel/blob/categorical/snorkel/learning/gen_learning.py#L231), since we aren't looking at the values the LFs give anymore.

Should I cut the number of total iterations down, or is this fast enough?